package.json

```diff
@@ -36,7 +36,7 @@
     "focus-visible": "^5.2.0",
     "fuse.js": "^6.6.2",
     "js-yaml": "^4.1.0",
-    "next": "~13.1.6",
+    "next": "~13.5.0",
```

Comment:
CI is failing with the following error:
```
Run yarn install --frozen-lockfile
yarn install v1.22.19
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
error An unexpected error occurred: "https://registry.yarnpkg.com/next/-/next-13.5.0.tgz: Request failed \"404 Not Found\"".
info If you think this is a bug, please open a bug report with the information provided in "/home/runner/work/devfile-web/devfile-web/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
Error: Process completed with exit code 1.
```

Next 13.5.0 does not seem to be available - see https://github.com/vercel/next.js/issues/55599
We'll need to bump to the latest Next 13.5.x instead..